src/sync/installation.ts

```diff
@@ -228,34 +228,28 @@ async function doProcessInstallation(app, data: BackfillMessagePayload, sentry:
 	const processor = tasks[task];
 
 	const execute = async () => {
-		if (await booleanFlag(BooleanFlags.SIMPLER_PROCESSOR, true)) {
```

Comment:
The feature flag has been serving `false` for quite some time now. That means we always executed the `else` branch.